README.md

```diff
@@ -0,0 +1,74 @@
+# Console for Apache Kafka™ on Kubernetes
+
+This project is a web console designed to facilitate interactions with Apache Kafka™ instances on Kubernetes, leveraging the [Strimzi](https://strimzi.io) Cluster Operator. 
+It is composed of two main parts: 
+
+- a REST API backend developed with Java Quarkus
+- a user interface (UI) built with Next.js and  [PatternFly](https://patternfly.org)
+
+### Getting started
+
+#### API
+
+Ensure you have Kubernetes and Strimzi Cluster Operator installed on your system.
+
+```bash
+cd api
+```
+
+Create a `.env` file containing the details about how the API should connect to the Kafka cluster.
+For a Kafka cluster installed using [Strimzi's Quick Starts](https://strimzi.io/quickstarts/) it should look like this:
+
+```.dotenv
+CONSOLE_KAFKA_MY_CLUSTER=kafka/my-cluster
+CONSOLE_KAFKA_MY_CLUSTER_BOOTSTRAP_SERVERS=my-cluster-kafka-bootstrap:9092
+```
+
+Then run the application. 
+
+```bash
+./mvn quarkus:dev -DskipTests
```

Comment:
this is `dev` mode and requires a bunch of dependencies, e.g. a compatible JDK installed globally, either we remove the requirement or we add a section `Pre-requisites` in this document.